isaric/parsers/uganda-vri.toml

```diff
@@ -0,0 +1,476 @@
+
+[[observation]]
+  name = "diastolic_blood_pressure_mmHg"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  value = { field = "diastol" }
+
+[[observation]]
+  name = "diastolic_blood_pressure_mmHg"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  value = { field = "diastol_iso" }
+
+[[observation]]
+  name = "glasgow_coma_score"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  value = { field = "gcs" }
+
+[[observation]]
+  name = "glasgow_coma_score"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  value = { field = "mental" }
+
+[[observation]]
+  name = "bleeding_haemorrhage"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  is_present = { field = "bleed", ref = "Y/N" }
+
+[[observation]]
+  name = "confusion"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  is_present = { field = "confused", ref = "Y/N" }
+
+[[observation]]
+  name = "cough"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  is_present = { field = "cof", ref = "Y/N" }
+
+[[observation]]
+  name = "cough"
+  date = { field = "hospitalization_date" }
+  phase = "admission"
+  is_present = { field = "sick_symp___2", values = { 1 = true } }
+
+[[observation]]
+  name = "diarrhoea"
+  date = { field = "hospitalization_date" }
+  phase = "admission"
+  is_present = { field = "sick_symp___3", values = { 1 = true } }
+
+[[observation]]
+  name = "diarrhoea"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  is_present = { field = "diarr", ref = "Y/N" }
+
+[[observation]]
+  name = "headache"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  is_present = { field = "head", ref = "Y/N" }
+
+[[observation]]
+  name = "history_of_fever"
+  date = { field = "hospitalization_date" }
+  phase = "admission"
+  is_present = { field = "sick_symp___1", values = { 1 = true } }
+
+[[observation]]
+  name = "history_of_fever"
+  date = { field = "vital_signs_date" }
+  phase = "study"
+  is_present = { field = "fever_hist", ref = "Y/N" }
+
+[[observation]] # CHECK THIS!
+  # 1 month prior to illness, did patient need help?
+  name = "inability_to_walk_scale"
+  date = { field = "hospitalization_date" }
+  phase = "admission"
+  text = { field = "move", values = { 1 = "1 - No difficulty", 2 = "2 - Some difficulty", 3 = "4 - Unable to walk" } }
+  context = ['walk around the home']
+
+[[observation]] # CHECK THIS!
+  name = "inability_to_walk_scale"
+  date = { field = "hospitalization_date" }
+  phase = "admission"
+  text = { field = "walk", values = { 1 = "1 - No difficulty", 2 = "2 - Some difficulty", 3 = "4 - Unable to walk" } }
+  context = ['walk on a flat surface for 20 mins']
+
+[[observation]] # CHECK THIS!
+  name = "inability_to_walk_scale"
+  date = { field = "hospitalization_date" }
+  phase = "admission"
+  text = { field = "stairs", values = { 1 = "1 - No difficulty", 2 = "2 - Some difficulty", 3 = "4 - Unable to walk" } }
+  context = ['walk up a hill']
```

Comment:
I'm not sure we need these first 3? They relate to 1 month prior to the onset of illness, i.e. pre-admission symptoms (really, pre-illness symptoms), which we wouldn't usually record. At the very least there should be a different date, as it's not relevant to their condition at admission.

Reply:
Happy to remove this! .. although there is option or allowance for _phase = 'pre-admission'_. I now wonder what we would use _phase = 'pre-admission'_ for ...

Reply:
thanks! @pipliggins - could you help with this? I couldn't figure it out - I wondered if this is a case for _combinedType_ ..?

Reply:
I believe we were only going to use pre-admission specifically for pre-admission meds (i.e. chronic medication) after a conversation with Laura et al - I think it's in the observations table rather than visit with the other meds so we can add that context. @abhidg have I got that right?

Do you mean for the other _walking_ fields? I can see a case for "max" combinedType, but we would want to weight it so that an inability to walk "around the home" is ranked as 'worse' than an inability to walk up a hill, (e.g. 4 for walking up a hill does not necessarily indicate a patient in worse heath than one who is a 3 for moving around the home). adtl doesn't have than functionality at the moment, and honestly I'm not sure we should introduce it just for this. I think either leave as-is, or just use the "walk around the home" variable, which I think is probably the closest to how this attribute is mapped in other studies.

Reply:
Pre-admission refers to strictly before admission, so symptoms at or before admission are classified as `admission`, whereas symptoms with a specific start date prior to admission should be `pre-admission`. This is only used in observations table as visit table only includes data for a admission.